Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstanding.cs

```diff
@@ -36,6 +36,9 @@ public enum ScanStates
         [Tooltip("Update period used after the scanning process is completed")]
         public float UpdatePeriod_AfterScanning = 4.0f;
 
+        public event OnScanDoneDelegate OnScanDone;
+        public delegate void OnScanDoneDelegate();
```

Comment:
Nit: I know the order really doesn't matter to the compiler but I think the delegate should be declared above the event.